src/mono/System.Private.CoreLib/src/System/RuntimeType.Mono.cs

```diff
@@ -1008,22 +1004,19 @@ public override MemberInfo[] GetMembers(BindingFlags bindingAttr)
         }
 
         [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicNestedTypes | DynamicallyAccessedMemberTypes.NonPublicNestedTypes)]
-        public override Type? GetNestedType(string fullname, BindingFlags bindingAttr)
+        public override Type? GetNestedType([MaybeNull] string name, BindingFlags bindingAttr)
```

Comment:
These are public overridden methods. Does `[MaybeNull]` impact how nullability of these methods is going to be seen via reflection?

Reply:
[Yes](https://github.com/dotnet/runtime/blob/d4c7d865558316da4c135f519eb3dfc25b8a4367/src/libraries/System.Private.CoreLib/src/System/Reflection/NullabilityInfoContext.cs#L156), but the fact that `name` might be reassigned to `null` is not meaningful because the parameter is passed by value.

Reply:
I would just copy it into a local variable.

Reply:
Reverted changes and kept status quo.